src/saltext/vmware/modules/esxi.py

```diff
@@ -4156,3 +4156,63 @@ def get_reference_schema():
     """
     log.debug("Running vmware_esxi.retrieve_reference_schema")
     return retrieve_reference_schema(Product.ESX)
+
+
+def pre_check(profile=None, cluster_paths=None, desired_state_spec=None, esx_config=None):
```

Comment:
As per the meeting today, this should be 'test=True' on remediation

Reply:
As per the demo given,and in the discussion we have Test=true logic moved to State Module and the changes have been incorporated accordingly